docs/guide/subscriptions.md

```diff
@@ -1,6 +1,136 @@
-### Subscriptions
+# Subscriptions
 
 Subscriptions are supported using the
 [Strawberry Django Channels](https://strawberry.rocks/docs/integrations/channels) integration.
 
-Check its docs to know how to use it.
+This guide will give you a minimal working example to get you going.
+There are 3 parts to this guide:
+
+1. Making Django compatible
+2. Setup local testing
+3. Creating your first subscription
+
+## Making Django compatible
+
+It's important to realise that Django doesnt support websockets out of the box.
+To resolve this, we can help the platform along a little.
+
+Edit your `MyProject.asgi.py` file and replace it with the following content.
+Ensure that you replace the relevant code with your setup.
```

Comment:
I think it is probably interesting to point out that [channels](https://channels.readthedocs.io/en/latest/) can be used to add websockets capabilities to django, and that the user can follow its instructions on how to setup it (as it will probably be more updated and have more content than us).

After that we can provide a minimal example to get started